src/index.html

```diff
@@ -10,24 +10,38 @@
     <div class="login-wrap">
       <form>
         <div class="field">
-          <input type="text" name="firstName"
-                 class="field-text" autocomplete="off"
-                 id="sign-up-name" autofocus>
+          <input
+            type="text"
+            name="firstName"
+            class="field-text"
+            autocomplete="off"
+            id="sign-up-name"
+            autofocus
+            placeholder="">
```

Comment:
Arrows in tags must be on the same line, see the example.
It needs to be fixed everywhere
```suggestion
          <input
            type="text"
            name="firstName"
            class="field-text"
            autocomplete="off"
            id="sign-up-name"
            autofocus
            placeholder=""
          >
```